config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -199,7 +197,7 @@
 #
 # In previous versions of Rails, Action View always used `Rails::HTML4::Sanitizer` as its vendor.
 #
-# Rails.application.config.action_view.sanitizer_vendor = Rails::HTML::Sanitizer.best_supported_vendor
+Rails.application.config.action_view.sanitizer_vendor = Rails::HTML::Sanitizer.best_supported_vendor
```

Comment:
I'm assuming that the santizer specs would catch any issues switching from an html4 to html5 sanitizer here ... may want to look more closely if we're not confident in that.

Reply:
I'm not sure which sanitizer specs you're referring to. I don't think we have any sanitizing tests except for the remote post sanitization code using the `sanitize` gem directly rather than Rails' sanitizing code.

Reply:
You are correct -- I had forgotten that whole implementation uses the gem and not the internals.  I think that makes this one safe since we're changing a config on something we don't use.